spec/models/course_user_spec.rb

```diff
@@ -147,6 +161,29 @@
       end
     end
 
+    describe '#level_progress_percentage' do
+      subject { student.level_progress_percentage }
+      before do
+        create :course_level, course: course, experience_points_threshold: 100
+        course.reload
+      end
+
+      it { should be_a(Fixnum) }
+      it { should be_between(0, 100) }
+      it 'returns the accurate value' do
```

Comment:
doesn't describe it as 'accurate' or 'correct', explain what the expected value is, i.e. 99%, within a context of 'when the user has 99% of the points required'